VoiceAttackResponder/VoiceAttackPlugin.cs

```diff
@@ -452,11 +455,17 @@ public static void InvokeEDDBSystem(ref dynamic vaProxy)
                 }
                 string systemUri = "https://eddb.io/system/" + EDDI.Instance.CurrentStarSystem.EDDBID;
 
-                Logging.Debug("Starting process with uri " + systemUri);
-
+                if (vaProxy.GetBoolean("EDDI open uri in browser") != false)
```

Comment:
I'm going to invoke the "Don't Repeat Yourself" principle here and extract a mthod for this.